ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -45,6 +45,7 @@ class ShyHeaderController: UIViewController {
     }
 
     private var accessoryViewObservation: NSKeyValueObservation?
+    private var wideAccessoryViewObservation: NSKeyValueObservation?
```

Comment:
I think I would prefer a name that better describes that the view will live under the main accessory view. Maybe `secondaryAccessoryView`?